gap-analysis/index.md

```diff
@@ -345,47 +345,11 @@ The <semantics> element may be used to attach content markup as an annotation to
 </semantics>
 ```
 
+The `<csymbol>point<csymbol/>` element in the above example shows one style of markup that can be used to refer to operators that are not part of standard content MathML. It is used here to illustrate the style of parallel markup elements that could be used to represent the point example introduced above.  Other strategies to handle references to symbols outside of MathML include URI-style attributes (`definitionURL`), and attributes that link into OpenMath content dictionaries (`cdbase` and `cd`).
+The OpenMath dictionaries do not currently associate speech with the meanings they define and so are not a good candidate to point to.
+A better approach would be to link to Wikidata as described in [this paper](http://ceur-ws.org/Vol-2307/paper51.pdf). For example, [the Wikidata definition of “point” is here](https://www.wikidata.org/wiki/Q44946); with the exception of `point-coordinate` (third example) all the other notations used in the examples including a $A'$ are already part of Wikidata.
```

Comment:
This `A better approach` contrast is strange, as wikidata doesn't associate speech either, it's just a whole lot bigger, and (thankfully) directly derived from natural language.

Reply:
How about replacing the paragraph after the first example with:
> Content MathML has a few methods to associate meaning with a `csymbol`. One method is to use one of the ~140 predefined symbols. This is referred to as Pragmatic Content MathML. Neither `point` nor the other `csymbol`s used in the examples are part of Pragmatic Content MathML. The `definitionURL` attribute of `csymbol` along with `cdbase` and `cd` (used with OpenMath content dictionaries) provides an open-ended way to attach a meaning to `csymbol`. For simplicity, those are omitted in the examples in this section. One possible source of definitions that might be able to be used to associate speech with a definition is Wikidata (see [this paper](http://ceur-ws.org/Vol-2307/paper51.pdf) for more information). For example, [the Wikidata definition of “point” is here](https://www.wikidata.org/wiki/Q44946).

I am being pretty wishy-washy about using Wikidata...

Reply:
I've just pushed a change deleting it. The chapter 4 intro mentions trying "to strike a more pragmatic balance between verbosity and formality", but you are right, that's the only mention of pragmatic. I changed the wording to stick with "predefined functions"